cloudfoundry-cli/parts/plugins/x-cf.py

```diff
@@ -0,0 +1,8 @@
+from snapcraft.plugins import go
+
+
+class CFPlugin(go.GoPlugin):
+
+    def _local_build(self):
+        self._run(['go', 'install', './{}/...'.format(
+            'github.com/cloudfoundry/cli/main')])
```

Comment:
It's not really a bug since there's no way to ask `go get` to use a tag, but yeah, you're right.